web-app/js/portal/details/SubsetItemsTabPanel.js

```diff
@@ -20,23 +20,9 @@ Portal.details.SubsetItemsTabPanel = Ext.extend(Ext.TabPanel, {
                 this.subsetPanel,
                 this.infoPanel,
                 this.layerDetailsPanel
-            ],
-            listeners: {
-                beforetabchange: this._doTracking
-            }
+            ]
         }, cfg);
 
         Portal.details.SubsetItemsTabPanel.superclass.constructor.call(this, config);
-    },
-
-    _doTracking: function(tabPanel, newTab, oldTab){
```

Comment:
Yeah. I'm guessing it was moved but the old code was left in place.
https://github.com/aodn/aodn-portal/blob/e0a74e1eba3ad0ee0da85b3299224cb3151768f7/web-app/js/portal/details/SubsetItemsWrapperPanel.js#L120